notifier/notifier.go

```diff
@@ -126,14 +105,26 @@ func New(s *storage.Storage, concurrency int, logger *log.Logger, dwnlURL string
 
 // Start starts the Notifier loop and instruments the worker goroutines that
 // perform the actual notify requests.
-func (n *Notifier) Start(closeChan chan struct{}) {
+func (n *Notifier) Start(closeChan chan struct{}, cfg map[string]interface{}) {
+	ctx, cancelfunc := context.WithCancel(context.Background())
+
+	for id, backend := range n.backends {
```

Comment:
here you are shadowing the `backend` package. Please use something like `for id, _backend`

Reply:
ah yes! great advice! thanks will fix!

Reply:
Use `b` for short loops where the type is apparent.